opensrp-app/src/main/java/org/smartregister/sync/helper/SyncSettingsServiceHelper.java

```diff
@@ -89,24 +89,22 @@ protected CoreLibrary getInstance() {
     }
 
     // will automatically use the resolve check
+
     private void getExtraSettings(JSONArray settings, String accessToken) throws JSONException {
         JSONArray completeExtraSettings = new JSONArray();
         if (getInstance().getSyncConfiguration().hasExtraSettingsSync()) {
-            List<String> syncParams = getInstance().getSyncConfiguration().getExtraSettingsParameters();
-            if (syncParams.size() > 0) {
-                for (String params : syncParams) {
-                    String url = SettingsSyncIntentService.SETTINGS_URL + "?" + params + "&" + AllConstants.SERVER_VERSION + "=" + sharedPreferences.fetchLastSettingsSyncTimeStamp() + "&" + AllConstants.RESOLVE + "=" + getInstance().getSyncConfiguration().resolveSettings();
-                    JSONArray extraSettings = pullSettings(url, accessToken);
-                    if (extraSettings != null) {
-                        aggregateSettings(completeExtraSettings, extraSettings);
-                    }
-                }
-
-                aggregateSettings(settings, completeExtraSettings);
+            String syncParams = getInstance().getSyncConfiguration().getExtraSettingsParameters();
+            String url = SettingsSyncIntentService.SETTINGS_URL + "?" + syncParams + "&" + AllConstants.SERVER_VERSION + "=0&" + AllConstants.RESOLVE + "=" + getInstance().getSyncConfiguration().resolveSettings();
```

Comment:
Also check out if the [Request Params Builder class](https://github.com/opensrp/opensrp-client-core/blob/ec663ec9c86beeea92819517f271f725a4bcc44f/opensrp-app/src/main/java/org/smartregister/sync/intent/BaseSyncIntentService.java#L37) in the `BaseSyncIntentService` would be helpful here